backend/core/serializers/user.py

```diff
@@ -21,26 +22,53 @@ def _validate_username(value):
     return value
 
 
+@wrap_validation_error_with_field("email")
+def validate_email_uniqueness_iexact(value: str) -> str:
+    """Raise an error if an email is considered already in use.
+
+    Even if emails are considered case-sensitive by the RFC 5321, several
+    email providers don't follow this specification. This validator ensures
+    users won't be able to create several accounts with the same address
+    by changing only the case.
+    """
+    try:
+        User.objects.get(email__iexact=value.lower())
```

Comment:
`.lower()` is redundant with "iexact", right ?
```suggestion
        User.objects.get(email__iexact=value)
```

Reply:
I just realized it too, it's indeed useless.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/812/commits/518bb64042d9a76db4f4e128cdf0146fe415484f.